test/utils.js

```diff
@@ -0,0 +1,79 @@
+const StreamrClient = require('streamr-client')
+const mqtt = require('async-mqtt')
+
+const createBroker = require('../src/broker')
+
+function startBroker(id, httpPort, wsPort, networkPort, trackerPort, mqttPort, enableCassandra, privateKeyFileName, certFileName) {
```

Comment:
Just an idea.

Would it make sense to group related parameters next to each other in list, e.g., non-optional, adapters, cassandra, certs. Makes it easier to remember.
Like
```js
startBroker(id, networkPort, trackerPort, wsPort, httpPort, mqttPort, enableCassandra, privateKeyFileName, certFileName)
```